book/command-line-parsing/README.md

```diff
@@ -215,11 +215,11 @@ And that's all it took to build our little MD5 utility! Here's a complete
 version of the example we just walked through, made slightly more succinct by
 removing intermediate variables.
 
-```ocaml file=examples/md5_succinct/md5.ml
+```ocaml file=examples/correct/md5_succinct/md5.ml
 open Core
 
 let do_hash file =
-  Md5.digest_file_blocking file
+  Md5.digest_file_blocking filee
```

Comment:
`filee` -> typo?

Reply:
Not a typo.  I broke one of the examples to see if the test would break, and it didn't.  I'm still trying to get this fixed.